### PR TITLE
feat(pattern): created an alpha version of the logo grid

### PR DESCRIPTION
**Pull request by mkothur**
### Related Ticket(s)

Related to issue #763 
### Description

Created the alpha version of a logo grid pattern
image

### CODE REVIEW

File: packages/styles/scss/patterns/logo-grid/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+#### Elements
+
+| Class                       | Description                                                                     |
+| --------------------------- | ------------------------------------------------------------------------------- |
+| `.bx--logo-grid__container` | Element containing all logo grid rows                                           |
+| `.bx--logo-grid__row`       | Horizontal divisions across the viewport                                        |
+| `.bx--logo-grid__col`       | Vertical divisions across the viewport and change in width at different screen. |  | `.bx--logo-grid__title` | Title for Logo Grid |
+| `.bx--logo-grid__wrapper`   | Wrapper for Logo placeholders                                                   |
+| `.bx--logo-grid__logo`      | Placeholder for logo grid                                                       |
```
Comment (jeffchew) [inline packages/styles/scss/patterns/logo-grid/README.md]: @mkothur the table here looks off for some reason?

Comment (mkothur) [reply packages/styles/scss/patterns/logo-grid/README.md]: Okay, @jeffchew let me fix.

File: packages/styles/scss/patterns/logo-grid/_logo-grid.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+    }
+    &__title {
+      margin: auto;
+      width: 45%;
```
Comment (jeffchew) [inline packages/styles/scss/patterns/logo-grid/_logo-grid.scss]: @mkothur This explicit width seems oddly specific. This is in the designs @wonilsuhibm ?

**Comment by jeffchew** [comment]
Alignments look a little weird, and it looks really off in mobile and tablet views...

cc: @wonilsuhibm 

Screen Shot 2019-12-31 at 1 10 50 PM
Screen Shot 2019-12-31 at 1 10 31 PM

File: packages/styles/scss/patterns/logo-grid/_logo-grid.scss (hunk 2 above)
Comment (mkothur) [reply packages/styles/scss/patterns/logo-grid/_logo-grid.scss]: Can I have the design specs for this pattern, Actually I took the reference from the ticket description.

**Comment by mkothur** [comment]
@jeffchew will fix them.

File: packages/styles/scss/patterns/logo-grid/_logo-grid.scss (hunk 2 above)
Comment (photodow) [reply packages/styles/scss/patterns/logo-grid/_logo-grid.scss]: I don't think we have design specs for this yet. I'm at the moment thinking based on where the design is that the title here would use the copy from the content section without its title. So close to what we currently have in the `ListSection` pattern.

image

File: packages/patterns-react/src/patterns/LogoGrid/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.';
+
+const logosGroup = [
+  {
+    title: 'PHILIPS',
+    imgSrc: 'https://via.placeholder.com/140',
+    altText: 'placehoder',
+  },
+  {
+    title: 'charles SCHWAB',
```
Comment (jeffchew) [inline packages/patterns-react/src/patterns/LogoGrid/README.md]: Slightly nitpicky, but not sure if we should mention actual companies in our open source code. You can probably use `IBM`, or `Company A` and `Company B`.

File: packages/patterns-react/src/patterns/LogoGrid/__stories__/LogoGrid.stories.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import { DDS_LOGO_GRID } from '../../../internal/FeatureFlags';
+import LogoGrid from '../LogoGrid';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+const logosGroup = [
+  {
+    label: 'PHILIPS',
```
Comment (jeffchew) [inline packages/patterns-react/src/patterns/LogoGrid/__stories__/LogoGrid.stories.js]: Change the company names here as well, probably shouldn't have actual company names in our code.

**Comment by mkothur** [comment]
@jeffchew: updated the PR for above changes.

**Pull request merged**